test/controllers/sign.out.controller.spec.ts

```diff
@@ -42,7 +42,7 @@ describe("Sign Out controller", () => {
   });
 
   describe("GET tests", () => {
-    test(`renders the ${config.SIGN_OUT_PAGE} page, when FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022 is active, with ${config.MANAGING_OFFICER_CORPORATE_PAGE} as back link`, async () => {
+    test(`renders the ${config.SIGN_OUT_PAGE} page, with ${config.MANAGING_OFFICER_CORPORATE_PAGE} as back link`, async () => {
```

Comment:
might be able to remove the mockIsActiveFeature.mockReturnValueOnce(true) here too. If it works, probably good idea to check the other tests that have changed and see if the mockIsActiveFeature can be removed in those too.

Reply:
Removed this.